fixtures/vcr_cassettes/okapi-cli-specs.yml

```diff
@@ -0,0 +1,72 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: https://okapi.frontside.io/_/proxy/modules
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      Accept-Encoding:
+      - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
+      Accept:
+      - "*/*"
+      User-Agent:
+      - Ruby
+  response:
+    status:
+      code: 200
+      message: OK
+    headers:
+      Server:
+      - nginx/1.13.3
+      Date:
+      - Thu, 05 Oct 2017 21:53:45 GMT
+      Content-Type:
+      - application/json
+      Transfer-Encoding:
+      - chunked
+      Connection:
+      - keep-alive
+      Strict-Transport-Security:
+      - max-age=15724800; includeSubDomains;
+    body:
+      encoding: ASCII-8BIT
+      string: |-
+        [ {
+          "id" : "folio-mod-configuration",
+          "name" : "Configuration"
+        }, {
+          "id" : "mod-kb-ebsco-7c3d2a6d65d7791a936ba754f72a133e891dbddf",
+          "name" : "kb-ebsco"
+        }, {
+          "id" : "folio-mod-auth-token",
+          "name" : "authtoken"
+        }, {
+          "id" : "folio-mod-circulation",
+          "name" : "Circulation Module"
+        }, {
+          "id" : "folio-mod-permissions",
+          "name" : "permissions"
+        }, {
+          "id" : "folio-mod-inventory-storage",
+          "name" : "Inventory Storage Module"
+        }, {
+          "id" : "folio-mod-login",
+          "name" : "login"
+        }, {
+          "id" : "folio-mod-users",
+          "name" : "users"
+        }, {
+          "id" : "folio-mod-inventory",
+          "name" : "Inventory Module"
+        }, {
+          "id" : "folio-mod-circulation-storage",
+          "name" : "Circulation Storage Module"
+        }, {
+          "id" : "folio-mod-users-bl",
+          "name" : "users business logic"
+        } ]
+    http_version: 
+  recorded_at: Thu, 05 Oct 2017 21:53:45 GMT
+recorded_with: VCR 3.0.3
```

Comment:
This uses the old okapi-1.5 cluster. We should probably use `https://okapi2.frontside.io` (or switch the new okapi-2.x cluster to `https://okapi.frontside.io`.

Particularly, _most_ internal endpoints will now complain with
```
No suitable module found for http://okapi2.frontside.io/_/proxy/modules
```
if you try to hit the endpoint with the `X-Okapi-Tenant` header set (unless it's set to `okapi.supertanent`)

Reply:
I'm totally ok using either one for the tests.

> if you try to hit the endpoint with the X-Okapi-Tenant header set (unless it's set to okapi.supertanent)

"with" or "without"

Reply:
With

![image](https://user-images.githubusercontent.com/5005153/31286359-87c6fd4c-aa83-11e7-92b8-f8fb890b7c13.png)

Reply:
Without the header is fine, but we have to make sure we don't add the tenant to these internal endpoints

![image](https://user-images.githubusercontent.com/5005153/31286422-b38cf49a-aa83-11e7-90bf-7425b99b3ed4.png)

Reply:
`okapi.supertenant` also works, so maybe instead of removing the tenant for those endpoints, we ensure that the tenant is the supertenant for those endpoints.

Reply:
I see, this is actually related to the change around settings too. It should "just work" In other words, i made the change to _not_ actually use either the tenant or the token unless you explicity ask for it. even if the `--tenant` or `OKAPI_TENANT` options are set.

```ruby
client.get('/_/proxy/modules') # does not set tenant header
client.tenant.get('/_/proxy/modules') # does set tenant header.
```

Reply:
I see. But `client.has_interface?('configuration')` will fail because there is no tenant for `/_/proxy/tenants/#{@settings.tenant}/interfaces/#{interface_name}`.

However `client.tenant.has_interface?('configuration')` will also fail because the request automatically sets the `x-okapi-tenant` header.